test/unit_LineSegment.jl

```diff
@@ -103,6 +103,31 @@ for N in [Float64, Rational{Int}, Float32]
     intersection_empty, point = is_intersection_empty(l1, l1_copy, true)
     @test !is_intersection_empty(l1, l1_copy) && !intersection_empty && point ∈ l1
 
+    # intersection
+    s1 = LineSegment(N[-5.0,-5.0], N[5.0,5.0])
+    # colinear shifted down
+    s2 = LineSegment(N[-6.0,-6.0], N[4.0,4.0])
+    @test intersection(s1, s2) isa LineSegment
+    @test isapprox(intersection(s1, s2).p, N[-5,-5])
+    @test isapprox(intersection(s1, s2).p, N[4,4])
```

Comment:
There is something wrong here. The second test fails.

Reply:
```suggestion
    @test isapprox(intersection(s1, s2).p, N[-5,-5])
    @test isapprox(intersection(s1, s2).q, N[4,4])
```
little typo :sweat_smile: